docs/content/modeling/getting-started.mdx

```diff
@@ -842,43 +842,49 @@ We can achieve that with the following definition using <UpdateProductNameInLink
 
 <AuthzModelSnippetViewer
   onlyShow={SyntaxFormat.Friendly2}
+  skipVersion={true}
   configuration={{
-    can_share: {
-      union: {
-        child: [
-          {
-            computedUserset: {
-              object: '',
-              relation: 'owner',
-            },
-          },
-          {
-            computedUserset: {
-              object: '',
-              relation: 'editor',
-            },
-          },
-          {
-            tupleToUserset: {
-              computedUserset: {
-                object: '',
-                relation: 'owner',
+    schema_version: '1.1',
+    type_definitions: [
```

Comment:
Can we make the rest like this? A valid config with the version at the model level?

Reply:
the rest of the examples in this page is like this.  Maybe can you point out the case that it isn't?

Reply:
I meant in the concepts page

Reply:
Done